extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
```

Comment:
Please consider changing method name. If I understand correct it should open a new window with infobox preview not editor preview